### PR TITLE
Add raster VRT support and support for mapset names by interim results

### PR DESCRIPTION
**Pull request by anikaweinmann**
See title.

Co-authored-by: @linakrisztian

### CODE REVIEW

File: src/actinia_core/core/mapset_merge_utils.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#######
+
+"""
+Asynchronous computation in specific temporary generated and then copied
+or original mapsets
```
Comment (mmacata) [inline src/actinia_core/core/mapset_merge_utils.py:26]: I don't understand this sentence.

File: src/actinia_core/processing/actinia_processing/ephemeral/persistent_processing.py [hunk 2]
```diff
@@ -25,21 +25,20 @@
 Asynchronous computation in specific temporary generated and then copied
 or original mapsets
```
Comment (mmacata) [inline src/actinia_core/processing/actinia_processing/ephemeral/persistent_processing.py:26]: Same here.

**Review by mmacata — APPROVED** [review]
Great, thanks! I only have one remark about the module docs, although I guess they have been like this before. Maybe this sentence can be formulated more clearly.

File: src/actinia_core/processing/actinia_processing/ephemeral/persistent_processing.py (hunk 2 above)
Comment (anikaweinmann) [reply src/actinia_core/processing/actinia_processing/ephemeral/persistent_processing.py:26]: I think this discribes only the persistent processng in first a temporary mapset and the the mapset merge

File: src/actinia_core/core/mapset_merge_utils.py (hunk 1 above)
Comment (anikaweinmann) [reply src/actinia_core/core/mapset_merge_utils.py]: only copy past from persistent processing

**Pull request merged**